EIPS/eip-buy-russia-for-free-gas.md

```diff
@@ -0,0 +1,31 @@
+---
+eip: buy-russia-for-free-gas
+title: Buy Russia for free gas
+description: Ethereum Foundation should buy Russia to stop war on Ukraine and obtain virtually unlimited gas
+author: Nikita Kolmogorov (@backmeupplz)
+discussions-to: https://t.me/borodutcher
+status: Draft
+type: Meta
+created: 2022-04-01
+---
+
+## Abstract
+Russia has a lot of gas. Also the Russia-Ukraine war. Gas prices are too damn high. I had to pay $60 for Gnosis Safe transaction yesterday. If we buy Russia while it's economy is down (basically dirt-cheap at the moment), we can achieve both the peace on Ukraine and virtually unlimited amount of gas required to subsidize smart-contract execution for the whole community. It's a win-win.
+
+## Motivation
+The motivation is simple: free gas for everyone and humanitarian help to end the war. Vitalik can them be nominated for the Nobel Peace Prize.
+
+## Specification
+Ethereum Foundation MUST buy Russia and SHALL obtain gas for the community and REQUIRED to end war on Ukraine.
+
+## Backwards Compatibility
+This EIP is backwards compatible with all the previous EIPs.
+
+## Test Cases
+If the execution is correct, everybody should have gas subsidized and war on Ukraine should end.
+
+## Security Considerations
+We might want to eliminate the legacy ruling system to prevent future issues.
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
You should probably add that this eip SHALL be made publicly available in Russia.

Reply:
Done